infinitest-lib/pom.xml

```diff
@@ -55,5 +55,23 @@
 			<artifactId>commons-lang</artifactId>
 			<version>2.6</version>
 		</dependency>
+		<dependency>
+			<groupId>com.google.auto.value</groupId>
+			<artifactId>auto-value</artifactId>
+			<version>${autovalue.version}</version>
+			<scope>provided</scope>
```

Comment:
provided? How is it working?
Do we have this dependency available in a p2 repository?